.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  printWidth: 120,
```

Comment:
Just an opinion, not a change request: while initially quite irritating, I've come to appreciate the a printWidth of 80 [as recommended in the Prettier docs](https://prettier.io/docs/en/options.html#print-width). It makes PR reviews easier by assuring that changes are on lines that have less complexity. This makes it easier to spot the differences.  It makes side-by-side diffs less likely to have word wrapping. It also makes scanning the code easier (there's a UX principle that says to put scannable content along only one axis to make visually searching for content easier. Think Instagram, not Pinterest). And my personal favorites, _more files open next to each other on my monitor_ 😍. Anyway, just a thought.